app/src/test/java/com/example/marvelapp/data/mapper/CharacterRealmMapperTest.kt

```diff
@@ -0,0 +1,94 @@
+package com.example.marvelapp.data.mapper
+
+import com.example.marvelapp.utils.CharactersConstantsUtils.AVAILABLE
+import com.example.marvelapp.utils.CharactersConstantsUtils.CHARACTER_ENTITY
+import com.example.marvelapp.utils.CharactersConstantsUtils.CHARACTER_REALM_ENTITY
+import com.example.marvelapp.utils.CharactersConstantsUtils.COLLECTIONURI
+import com.example.marvelapp.utils.CharactersConstantsUtils.DESCRIPTION
+import com.example.marvelapp.utils.CharactersConstantsUtils.EXTENSION
+import com.example.marvelapp.utils.CharactersConstantsUtils.ID
+import com.example.marvelapp.utils.CharactersConstantsUtils.MODIFIED
+import com.example.marvelapp.utils.CharactersConstantsUtils.NAME
+import com.example.marvelapp.utils.CharactersConstantsUtils.PATH
+import com.example.marvelapp.utils.CharactersConstantsUtils.RESOURCEURI
+import com.example.marvelapp.utils.CharactersConstantsUtils.RETURNED
+import com.example.marvelapp.utils.CharactersConstantsUtils.URL_LIST_ENTITY
+import com.example.marvelapp.utils.CharactersConstantsUtils.URL_REALM_LIST_ENTITY
+import com.example.marvelapp.utils.CharactersConstantsUtils.ZERO_VALUE
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class CharacterRealmMapperTest {
+
+    @Test
+    fun transformWhenValidValuesTest() {
+        assertForValidValuesEntityToRealm()
```

Comment:
this call is unnecessary you can put all the code of that method here as you are not reusing it